lib/librato/rails/subscribers/job.rb

```diff
@@ -0,0 +1,15 @@
+module Librato
+  module Rails
+    module Subscribers
+      hooks = %w{enqueue_at enqueue perform_start perform}
+
+      hooks.each do |hook|
+        ActiveSupport::Notifications.subscribe "#{hook}.active_job" do |*args|
+          collector.group 'rails.job' do |c|
+            c.increment hook
```

Comment:
@akahn mentioned this: https://github.com/rails/rails/blob/master/activejob/lib/active_job/logging.rb#L24

Seems like we could initialize an `ActiveSupport::Notifications::Event` and use `#duration` to add timing, e.g. `rails.job.perform.time`. Thoughts?